Initial cookiecutter setup

selvakumars2 opened the pull request:
Initial cookiecutter setup files for baremetal-network-provisioning
repository

hp-networking commented:
.coveragerc has references to networking_vsphere.  Please fix that.

In doc/source/conf.py - Fix Copyright to 2014, Openstack Foundation 

In some files you have used 'baremetal_network_provisioning'.  In some other files you have used
'baremetal-network-provisioning'.  Please double check all of them.

In general observe openstack/networking-vsphere equivalent files once - for eg., files setup.cfg, requirements.txt, test-requirements.txt, tox.ini etc [ as looking out networking-l2gw alone might not be sufficient ].

selvakumars2 commented:
Hi Vivek,
thanks for reviewing the initial cookie cutter , I have addressed the review comments for .coverage & conf.py and also tested py27 , pep8 and docs and working fine.

let me know what you think !

hp-networking commented:
Thanks for addressing the review comments. Merging in the initial cookiecutter...
